snippets/gift-card-recipient-form.liquid

```diff
@@ -74,6 +74,15 @@
         {%- endif -%}
       </ul>
     </div>
+    <p
+      id="Recipient-fields-live-region-{{ section.id }}"
+      class="visually-hidden"
+      aria-live="polite"
+      aria-hidden="true"
```

Comment:
Here I think we could potentially approach it differently. What you have now is a toggling between hidden and not hidden. 
We could instead, have this `p` always there, visually hidden the way you have it and use the JS, not to toggle `aria-hidden`, but to inject the text. 

Because it's a `role="status"`, anytime something is added in that container, it will announce it if I'm not mistaking. 

For the strings, you can add them in `theme.liquid` like we do for: 
```javascript
      window.accessibilityStrings = {
        imageAvailable: `{{ 'products.product.media.image_available' | t: index: '[index]' }}`,
        shareSuccess: `{{ 'general.share.success_message' | t }}`,
        pauseSlideshow: `{{ 'sections.slideshow.pause_slideshow' | t }}`,
        playSlideshow: `{{ 'sections.slideshow.play_slideshow' | t }}`,
      };
```

This way it follows the pattern we've used in other part of the theme (eg: SKU on product page)